source/Ship.h

```diff
@@ -58,6 +58,15 @@ class Visual;
 // limits of what the AI knows how to command them to do.
 class Ship : public Body, public std::enable_shared_from_this<Ship> {
 public:
+	enum HyperStage {
+		NotJumping,
+		BeginJumping,
+		Jumping,
+		ChangeSystem,
+		InSystem,
+		EndJump
+	};
```

Comment:
if you still want the int thing you can make them correspond to ints, you dont have to ofc
for instance NotJumping could be 0 and the rest could be 1, 2...

(done by adding : int, look at VsyncState in preferencepanel.h for example)